specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.1/keys.json

```diff
@@ -1410,6 +1457,58 @@
       },
       "description": "As of http://tools.ietf.org/html/draft-ietf-jose-json-web-key-18"
     },
+    "KeyAttributesWithAttestation": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/KeyAttributes"
+        }
+      ],
+      "type": "object",
+      "properties": {
+        "attestation": {
+          "$ref": "#/definitions/KeyAttestation",
+          "description": "The key or key version attestation."
+        }
+      },
+      "description": "The attributes of a key managed by the key vault service."
+    },
+    "KeyAttestation": {
+      "type": "object",
+      "properties": {
+        "certificates": {
+          "type": "string",
+          "format": "base64url",
+          "description": "This is a base64 url encoded certificate bundle. This bundle contains certificates used for validating key attestation blob."
+        },
+        "firmwareVersion": {
+          "$ref": "#/definitions/FirmwareVersion",
+          "description": "The firmware version of the HSM."
+        },
+        "privateKeyAttestation": {
+          "type": "string",
+          "format": "base64url",
+          "description": "The attestation blob corresponding to private key."
+        },
+        "publicKeyAttestation": {
+          "type": "string",
+          "format": "base64url",
+          "description": "The attestation blob corresponding to public key in case of an asymmetric key."
+        }
+      }
+    },
+    "FirmwareVersion": {
+      "type": "object",
+      "properties": {
+        "major": {
+          "type": "string",
+          "description": "The major version of firmware."
+        },
+        "minor": {
+          "type": "string",
+          "description": "The minor version of firmware."
+        }
```

Comment:
Why are these strings? The fact there are separate major and minor components would seem to elude to them being integers. If both are strings, why not a single string?